blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,160 @@
+    channelDescription,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      ...(channelTitle && { title: channelTitle }),
+      link: `${domain}`,
+      ...(channelDescription && { description: channelDescription }),
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let category
+        const enclosureurl = jmespath.search(s, audioAvailable)
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        return {
+          title: jmespath.search(s, itemTitle) || '',
+          link: url,
+          guid: s._id,
+          pubDate: s[pubDate],
+          ...(enclosureurl && {
+            enclosure: {
+              '@url': enclosureurl,
+              '@type': getURLType(enclosureurl).type,
+            },
+          }),
+          ...(itemCategory &&
+            (category = jmespath.search(s, itemCategory)) &&
+            category && { category: category }),
+          description: cheerio
+            .load(
+              rssBuildContent.parse(
+                s.content_elements || [],
+                includeContent,
+                domain,
+                resizerKey,
+                resizerURL,
+                resizerWidth,
+                resizerHeight,
+              ) || '',
+            )
+            .text(),
+        }
+      }),
+    },
+  },
+})
+
+export function Rss({ globalContent, customFields, arcSite }) {
+  const {
+    resizerURL = '',
+    feedDomainURL = '',
+    feedTitle = '',
+    feedLanguage = '',
+  } = getProperties(arcSite)
+
+  const { width = 0, height = 0 } = customFields.resizerKVP || {}
+
+  const rssBuildContent = new BuildContent()
+
+  // can't return null for xml return type, must return valid xml template
+  return rssTemplate(get(globalContent, 'content_elements', []), {
+    ...customFields,
+    resizerURL,
+    resizerWidth: width,
+    resizerHeight: height,
+    domain: feedDomainURL,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  })
+}
+
+Rss.propTypes = {
+  customFields: PropTypes.shape({
+    audioAvailable: PropTypes.string.tag({
+      description: 'contains audio URL that alexa plays',
```

Comment:
You removed label.  Add it back with 'Audio URL'
You removed group. Add it back with 'Enclosure'
For description: 'ANS field to use for audio enclosure URL, if blank will be excluded.'